test-runner-frontend/src/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import reportWebVitals from './reportWebVitals';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  BrowserRouter,
+  Routes,
+} from "react-router-dom";
+import { Dashboard } from './Views/Dashboard';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+ /* <React.StrictMode>
+    <App />
+ </React.StrictMode>
+ */
+<React.StrictMode>
+  <BrowserRouter>
+    <Routes>
+      <Route path="/graphql-java-test-runner" element={<Dashboard/>}/>
```

Comment:
does this mean the dashboard url will be https://adarsh-jaiswal.github.io/graphql-java-test-runner? do you know if we need to look at using the basename prop?

Reply:
Yes, that will be the url, and I've already added the basename prop.